### ISSUE TITLE
drq version 01.00.28 seems to be broken

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#331]
The new version of `drq,` version 01.00.28, is broken: as soon the `--xls` option is specified in order to write the data request file I get the error bellow.

I am afraid I missed this until now, as the error message was among all the other output, and probably the old 01.00.27 data request files were still in place (and because the error does not appear when one asks the same request without the `--xls` option). I checked that the same drq commands with version  01.00.27 still work fine.

Failing run example:
`````
drq -m CMIP -t 1 -p 1 -e 1pctCO2 --xls --xlsDir cmip6-data-request-m=CMIP-e=1pctCO2-t=1-p=1
`````
The Error:
`````
INFO.volsum.01002: Writing cmip6-data-request-m=CMIP-e=1pctCO2-t=1-p=1/requestVol_CMIP_1_1.xlsx
Traceback (most recent call last):
  File "/home/thomas/anaconda2/envs/ece2cmor3/bin/drq", line 11, in <module>
    sys.exit(main_entry())
  File "/home/thomas/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/dreqPy/dreqCmdl.py", line 53, in main_entry
    x.run()
  File "/home/thomas/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/dreqPy/scope.py", line 1963, in run
    vs.anal(olab=mlab,doUnique=False, mode=vsmode, makeTabs=makeXls)
  File "/home/thomas/anaconda2/envs/ece2cmor3/lib/python2.7/site-packages/dreqPy/volsum.py", line 296, in anal
    self.accPdict[e].add( (rqv.vid,rqv.priority) )
AttributeError: 'vsum' object has no attribute 'accPdict'
`````

**Comment by treerink** [comment]
Hmm I see I have been producing xlsx files for CDRMIP before, as this is only available in 01.00.28 it looks like it has worked before. So lets check if my system is mixed up.

Can anybody quickly check for me if for instance this works:
`````
drq -m CMIP -e CMIP --xls
`````
with 
`````
drq-v
`````
you can see whether you are using version 01.00.28.

**Comment by klauswyser** [comment]
I just updated to dreqPy version 01.00.28 [Version 01.00.28], and indeed, the drq command then fails.

**Comment by treerink** [comment]
Thanks Klaus for checking.

**Comment by treerink** [comment]
I have recreated my ece2cmor3 environment by a full clean (conda clean --all -y), but the error remains.

I compared the behavior of `drq` version 01.00.28 when installed by the `ece2cmor3` environment and when it is installed just only directly in the anaconda shell (with pip, as we did before), but I don't find differences.

In fact there are some .xlsx files produced, and they have the same content in both cases. I diff-ed them with a xlsx diff tool:
`````
excel-diff xls-in-anaconda-shell-v28/cmvmm_CMIP_CMIP_1_1.xlsx xls-in-ece2cmor3-shell-v28/cmvmm_CMIP_CMIP_1_1.xlsx
excel-diff xls-in-anaconda-shell-v28/cmvmm_CMIP_TOTAL_1_1.xlsx xls-in-ece2cmor3-shell-v28/cmvmm_CMIP_TOTAL_1_1.xlsx
excel-diff xls-in-anaconda-shell-v28/requestVol_CMIP_1_1.xlsx xls-in-ece2cmor3-shell-v28/requestVol_CMIP_1_1.xlsx
`````

**Comment by treerink** [comment]
I also compared in the anaconda shell (thus using the directly the  pip install dreqPy==01.00.28 ) the differences between version 01.00.27 and 01.00.28:
`````
diff -r xls-in-anaconda-shell-v27 xls-in-anaconda-shell-v28
Only in xls-in-anaconda-shell-v27: cmvme_CMIP_1pctCO2_1_1.xlsx
Only in xls-in-anaconda-shell-v27: cmvme_CMIP_abrupt-4xCO2_1_1.xlsx
Only in xls-in-anaconda-shell-v27: cmvme_CMIP_amip_1_1.xlsx
Only in xls-in-anaconda-shell-v27: cmvme_CMIP_historical_1_1.xlsx
Only in xls-in-anaconda-shell-v27: cmvme_CMIP_piControl_1_1.xlsx
Binary files xls-in-anaconda-shell-v27/cmvmm_CMIP_CMIP_1_1.xlsx and xls-in-anaconda-shell-v28/cmvmm_CMIP_CMIP_1_1.xlsx differ
Binary files xls-in-anaconda-shell-v27/cmvmm_CMIP_TOTAL_1_1.xlsx and xls-in-anaconda-shell-v28/cmvmm_CMIP_TOTAL_1_1.xlsx differ
Binary files xls-in-anaconda-shell-v27/requestVol_CMIP_1_1.xlsx and xls-in-anaconda-shell-v28/requestVol_CMIP_1_1.xlsx differ
`````
This shows the MIP experiment request miss due to the error, but I also checked the files which are produced in both situations with excel-diff, and they do really differ.

From this it looks like we are missing all "experiment" requests, i.e. the `cmvme_*` files.

So I am afraid, **we are in trouble**.

But how could I have CDRMIP  `cmvme_*` files at all? I checked in the `ece2cmor3` shell:
`````
drq -m CDRMIP -e 1pctCO2-cdr --xls
`````
This one does work!

So obviously in some cases it goes wrong for some MIPs but not for all.

**Comment by treerink** [comment]
I sent an email to Martin Juckes, I hope we can solve this soon, because it mixes up everything.

**Comment by treerink** [comment]
The BUG is indeed reported with the same error message here: https://github.com/cmip6dr/dreqPy/issues/21
And upcoming data request version 01.00.29 will fix it. In the current beta release it is just fixed by Martin Juckes. I have to test it myself now.

**Comment by treerink** [comment]
The examples which did and did not work above, do indeed work with the beta 01.00.29  data request version installed by
`````
pip install  dreqPy==01.00.29b1
`````

**Comment by treerink** [comment]
Martin mailed that there will be few minor further changes for data request version 01.00.29 in the python code, and in the content. And that it will be hopefully out early next week.

**Comment by treerink** [comment]
The trouble with data request version 01.00.28 is solved by changing to the later version 01.00.29. With  data request version 01.00.29 `drq` works as before. In 2d56de1 we changed to data request version 01.00.29, i.e. `drq` version 01.00.29 is used in all the scripts.

However, the identification has not yet repeated with data request version 01.00.29, which is subject of #346.

So closing this issue.
